app/controllers/samples_controller.rb

```diff
@@ -156,6 +156,7 @@ def bulk_upload
 
     respond_to do |format|
       if @errors.empty? && !@samples.empty?
+        MetricUtil.put_metric_now("samples.uploaded", @samples.count)
```

Comment:
What is the difference between this upload and the one below?
Do metrics allow some kind of context or attributes to differentiate the two? Or would it have to be a different metric?

Reply:
This was in the bulk upload so it could be like 10 samples uploaded at a time.

Yes, you can add tags to the metrics (under the hood it counts as multiple metrics) https://docs.datadoghq.com/api/?lang=python#post-timeseries-points I plan to add the tags gradually for things like user_id